config/dev.exs

```diff
@@ -5,8 +5,5 @@ config :maru, Aprb.Api.Root,
 
 config :aprb, Aprb.Repo,
   adapter: Ecto.Adapters.Postgres,
-  username: System.get_env("DB_USER"),
-  password: System.get_env("DB_PASSWORD"),
-  database: System.get_env("DB_NAME"),
```

Comment:
Thanks! I think it makes more sense to leave `user`, `password`. Update `host` to have default of `loalhost` and hardcode database name. Something like:
```
   username: System.get_env("DB_USER"),
   password: System.get_env("DB_PASSWORD"),
   hostname: System.get_env("DB_HOST") || "localhost",
   database: "aprb_dev",
```
This way it should work on both Hokusai case and local. What do you think?

Reply:
Ok, I _think_ I got this how you wanted it with 66069d4. Let me know if I missed something!!